src/libraries/Native/Unix/System.Security.Cryptography.Native.Android/pal_signature.c

```diff
@@ -0,0 +1,65 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#include "pal_signature.h"
+
+int32_t AndroidCryptoNative_SignWithSignatureObject(JNIEnv* env, jobject signatureObject, jobject privateKey, const uint8_t* dgst, int32_t dgstlen, uint8_t* sig, int32_t* siglen)
```

Comment:
Have the android files been generally style-normalized with `clang-format -i`?

Reply:
I've touched a few of them with it as I've edited them, but there's been no consistent requirement to do so.

I think we should add something on the backlog to enforce formatting requirements/provide a tool to format all of the source.

Reply:
No worries.  If this was the odd man out then cleanup would be good.  For the existing libraries it used to be the case they were always formatted with `clang-format -i` with the default rules.  I don't know that anyone liked the rules, but they were consistent.

When I finish up the work on the OpenSSL version I'll probably have a PR that is just "formatting all the files again to return to a normalized state", since we've drifted off of that a bit.